Framework/PythonInterface/plugins/algorithms/GroupBySampleChangerPosition.py

```diff
@@ -0,0 +1,83 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2022 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+# pylint: disable=invalid-name,too-many-instance-attributes,too-many-branches,no-init,deprecated-module
+from mantid.simpleapi import GroupWorkspaces
+from mantid.api import (Algorithm, AlgorithmFactory, WorkspaceGroupProperty)
+from mantid.kernel import (Direction, StringArrayProperty)
+
+SAMP_POSITION = {6.0: "btm",
+                 85.5: "mid",
+                 165.0: "top",
+                 "None": "None"}
```

Comment:
Was a decision made about whether these should be hardcoded or not?

Reply:
as @SpencerHowells the sample change has been set up so these values are fine to be hard coded.